src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -52,6 +58,22 @@ public function addEntryFormAction(Request $request)
     /**
      * @param Request $request
      *
+     * @Route("/w", name="new_bookmarklet")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function addEntryViaBookmarklet(Request $request)
```

Comment:
You might need a test for that action.
And shouldn't this method be `addEntryViaBookmarkletAction` instead?

And why `/w` ?